Source/player.h

```diff
@@ -346,7 +346,7 @@ struct Player {
 	int8_t _pArmorClass;
 	int8_t _pMagResist;
 	int8_t _pFireResist;
-	int8_t _pLghtResist;
+	int8_t resistLightning; // _pLghtResist

```

Comment:
```suggestion
	int8_t resistLightning;
```
If this is also the case for all the other PRs then feel free to update them also.